lib/CFRPageActions.jsm

```diff
@@ -184,6 +184,16 @@ class PageAction {
    */
   async getStrings(string, subAttribute = "") {
     if (!string.string_id) {
+      if (subAttribute) {
+        return string.attributes[subAttribute];
+      }
+
+      if (string.attributes) {
```

Comment:
This check might want to be something with `string.value` maybe typeof string? Unless we're saying only set `.value` if specifying attributes because otherwise just use a plain string instead of an object.

Although I suppose additionally accepting an object with a single `value` property makes it simpler to remove attributes from a string… (but would that even happen anyway?)

Reply:
We would always specify an object, that can be `{string_id...}` or `{value...}` regardless of attributes.

Reply:
> only set .value if specifying attributes because otherwise just use a plain string instead of an object.

I think it would be: set .value or .string_id if specifying attributes otherwise use a plain string.